src/index.js

```diff
@@ -1,38 +1,80 @@
 import './styles/style.css';
+import {
```

Comment:
[OPTIONAL] Kindly add `"start": "webpack serve --open"` in scripts in the `package.json` file in order to be easier to test.